docs/cloud/csc_notebooks/guide_for_teachers.md

```diff
@@ -26,7 +26,7 @@ Instead of a course you can also use CSC Notebooks for collaboration. The workfl
 
 * The easiest is to use an existing Docker image, look these repositories for suitable images:
     * [Docker image sources in notebooks-public-images repository.](https://github.com/CSCfi/notebooks-public-images)
-    * [Rocker images](https://hub.docker.com/u/rocker) for different RStudio set-ups.
```

Comment:
To me it looks like Rocker project still hosts images on DockerHub as the primary source, and the other provided registry is GitHub container registry. 

https://rocker-project.org/images/versioned/r-ver.html#quick-reference